go/ct/gen/assignment_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package gen
+
+import (
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/ct/common"
+)
+
+func TestAssignment_String(t *testing.T) {
+
+	tests := map[string]struct {
+		value common.U256
+		want  string
```

Comment:
nit: since this test does not need to test the `String` function of U256 as well, all you would need  is the `value`. The `want` field in the struct can be derived int the test loop by calling `String()` on the value.